Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM openjdk:11 as build
+
+COPY gradle /srv/demo-api/gradle/
+COPY gradlew *.gradle.kts /srv/demo-api/
+COPY src /srv/demo-api/src/
+
+WORKDIR /srv/demo-api
+
+RUN ./gradlew clean build -x test
+
+
+FROM azul/zulu-openjdk-alpine:11.0.7-jre
+
+COPY --from=build /srv/demo-api/build/libs/demo-api.jar /
+
+CMD java -jar /demo-api.jar
+#
```

Comment:
Are this comments on some purpose?

Reply:
Yes. I will demo GraalVM during the talk ;)